rework_pysatl_mpest/preprocessing/components_family/classifier_criterions.py

```diff
@@ -0,0 +1,73 @@
+"""Module which contains collector of a vector of criterions for mixture classifier"""
+
+__author__ = "Mark Dubrovchenko"
+__copyright__ = "Copyright (c) 2025 PySATL project"
+__license__ = "SPDX-License-Identifier: MIT"
+
+import warnings
+from math import ceil
+
+import numpy as np
+from rework_pysatl_mpest.preprocessing.components_family.criterions import base_criterions
```

Comment:
The project follows a relative import style for internal modules.
```suggestion
from ..criterions import base_criterions
```